awx/main/management/commands/expire_sessions.py

```diff
@@ -29,9 +29,9 @@ def handle(self, *args, **options):
         # with consideration for timezones.
         start = timezone.now()
         sessions = Session.objects.filter(expire_date__gte=start).iterator()
-        request = HttpRequest()
         for session in sessions:
             user_id = session.get_decoded().get('_auth_user_id')
             if (user is None) or (user_id and user.id == int(user_id)):
-                request.session = import_module(settings.SESSION_ENGINE).SessionStore(session.session_key)
-                logout(request)
```

Comment:
Do we still need the `logout` import above?

Reply:
I reckon not.  There's a couple of other unused imports that I see now that I left lying around.

Reply:
Should be fixed now.